src/store.js

```diff
@@ -0,0 +1,48 @@
+import CoreObject from './metal/core-object';
+
+const Store = CoreObject.extend({
+  constructor() {
+    this.localStorageAvailable = this.storageAvailable('localStorage');
+    this.cache = {};
+  },
+
+  setItem(key, value) {
+    if (this.localStorageAvailable) {
+      localStorage.setItem(key, JSON.stringify(value));
+    } else {
+      this.cache[key] = value;
+    }
+
+    return value;
+  },
+
+  getItem(key) {
+    if (this.localStorageAvailable) {
+      const stringValue = localStorage.getItem(key);
+
+      try {
+        return JSON.parse(stringValue);
+      } catch (e) {
+        return null;
+      }
+    } else {
+      return this.cache[key] || null;
```

Comment:
I believe this would return `null` if the value is explicitly `false`. Unlikely to be an issue, but just in case...